dev/Project.toml

```diff
@@ -0,0 +1,9 @@
+[deps]
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+TimeZones = "f269a46b-ccf7-5d73-abea-4c690281aa53"
+
+[compat]
+JSON = "0.20.1, 0.21"
+HTTP = "0.8.1"
+julia = "1.3"
```

Comment:
```suggestion
julia = "1.3"

```

Reply:
I updated the commit where this was original made